node/buildpack.go

```diff
@@ -62,7 +64,32 @@ func (*NodeBuildpack) Build(b build.Build) error {
 	} else if !ok {
 		return fmt.Errorf("buildpack passed detection but did not know how to actually build")
 	}
-	return invoker.Contribute()
+	if err := invoker.Contribute(); err != nil {
+		return err
+	}
+
+	log, err := logger.DefaultLogger(b.Platform.Root)
+	if err != nil {
+		return err
+	}
+
+	layersDir := layers.NewLayers(b.Layers.Root, log)
+	systemLayer := layersDir.Layer("system")
+	sysFunc, err := NewSystemFunction(systemLayer)
+	if err != nil {
+		return err
+	} else if !ok {
+		return fmt.Errorf("buildpack passed detection but did not know how to actually build")
+	}
+
+	if err := sysFunc.Contribute(); err != nil {
+		return err
+	}
+
+	// make Ben happy (we can remove this when we remove libcfbuildpack)
```

Comment:
i don't care if Ben is happy :) do we need this still?

Reply:
I thought you added this? My original commit to this branch has your name on it now, so I assumed you rebased it.

Reply:
Or did this come from @hone?

Reply:
(I don't know who ben is or what makes him happy)

Reply:
oh i think it get it. Ben is the author of libcfbuildpack, and I think it does something weird if the layer hasn't been touched (like never actually created it)